message_definitions/v1.0/messages.xml

```diff
@@ -1850,7 +1870,7 @@
       <field name="overrun_cnt" type="uint16"/>
       <field name="noise_err_cnt" type="uint16"/>
       <field name="framing_err_cnt" type="uint16"/>
-      <field name="bus_number" type="uint8"/>
+      <field name="bus_number" type="uint8"/></field>
```

Comment:
this is wrong formatting here, field was already closed with `/>`

Reply:
Sorry this slipped through